frontend/src/metabase/query_builder/components/view/sidebars/QuestionInfoSidebar.tsx

```diff
@@ -28,10 +35,23 @@ export const QuestionInfoSidebar = ({ question, onSave }: Props) => {
     }
   };
 
+  const handleUpdateCacheTTL = (cache_ttl: number | undefined) => {
+    if (question.cache_ttl() !== cache_ttl) {
+      return onSave({
```

Comment:
I probably missed it before, but we should have methods on `Question` for updating the inner card that return a new instance. Like this https://github.com/metabase/metabase/blob/5bdde140a7b9165ebbedab07e301876589e0bcd8/frontend/src/metabase-lib/lib/Question.ts#L826